src/test/java/uk/gov/hmcts/reform/pcqbackend/repository/impl/ProtectedCharacteristicsRepositoryImplTest.java

```diff
@@ -173,8 +173,8 @@ void shouldFindAllByCaseIdNotNullAndLastUpdatedTimestampBefore() {
         Timestamp lastUpdatedTimestamp = new Timestamp(System.currentTimeMillis());
 
         // get record
-        final List<ProtectedCharacteristics> pc = protectedCharacteristicsRepository
-            .findAllByCaseIdNotNullAndLastUpdatedTimestampBefore(lastUpdatedTimestamp);
+        final List<String> pc = protectedCharacteristicsRepository
+            .findAllPcqIdsByCaseIdNotNullAndLastUpdatedTimestampBeforeWithLimit(lastUpdatedTimestamp, 1000);
 
         assertThat(pc).isNotEmpty();
```

Comment:
I know this is not your code but can you please improve assertion by verifying the total number of records.